target_snowflake/upload_clients/s3_upload_client.py

```diff
@@ -50,9 +50,10 @@ def upload_file(self, file, stream, temp_dir=None):
         bucket = self.connection_config['s3_bucket']
         s3_acl = self.connection_config.get('s3_acl')
         s3_key_prefix = self.connection_config.get('s3_key_prefix', '')
-        s3_key = "{}pipelinewise_{}_{}.csv".format(s3_key_prefix,
-                                                   stream,
-                                                   datetime.datetime.now().strftime("%Y%m%d-%H%M%S-%f"))
+        s3_key = "{}pipelinewise_{}_{}_{}".format(s3_key_prefix,
```

Comment:
Can you share an example of how this s3_key will look like with change?

Reply:
The pattern is `<s3_key_prefix>pipelinewise_<stream>-<timestamp>-batch-<unique_temp_file_id>.<file_format>[.<compression>`

With real examples, with all the 4 different file types:
```
snowflake-imports-test/pipelinewise_performance_test_20210309-101648-516712_batch_762pt8p5.parquet.gz
snowflake-imports-test/pipelinewise_performance_test_20210309-101814-465401_batch_jrl2tdl8.parquet
snowflake-imports-test/pipelinewise_performance_test_20210309-101855-026878_batch_keu6f7w8.csv.gz
snowflake-imports-test/pipelinewise_performance_test_20210309-102006-808052_batch_1qz9tbmn.csv
```

```

Reply:
Thanks.